lib/Backgroundjobs/Cleanup.php

```diff
@@ -13,15 +13,8 @@
 use OCP\IDBConnection;
 
 class Cleanup extends TimedJob {
-	/** @var IDBConnection */
-	private $db;
-	/** @var WopiMapper $wopiMapper */
-	private $wopiMapper;
-
-	public function __construct(ITimeFactory $time, IDBConnection $db, WopiMapper $wopiMapper) {
+	public function __construct(ITimeFactory $time, private IDBConnection $db, private WopiMapper $wopiMapper) {
```

Comment:
those should go then one parameter per line.

Reply:
Applied with the new php-cs-fixer config as it has https://github.com/kubawerlos/php-cs-fixer-custom-fixers#MultilinePromotedPropertiesFixer included